lib/resources/accounts.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const { normalizeNotes } = require('../utils/razorpay-utils')
+
+module.exports = function (api) {
+
+    const BASE_URL = "/accounts";
+
+    return {
+        create(params, callback) {
+            let { notes, ...rest } = params
+            let data = Object.assign(rest, normalizeNotes(notes));
+
+            return api.post({
+                version: 'v2',
+                url: `${BASE_URL}`,
+                data: data
+            }, callback);
+        },
+
+        edit(accountId, params, callback) {
+            let { notes, ...rest } = params
+            let data = Object.assign(rest, normalizeNotes(notes));
+
+            return api.patch({
+                version: 'v2',
+                url: `${BASE_URL}/${accountId}`,
+                data: data
+            }, callback);
+        },
+
+        fetch(accountId, callback) {
+            return api.get({
+                version: 'v2',
+                url: `${BASE_URL}/${accountId}`,
+            }, callback);
+        },
+
+        delete(accountId, callback) {
+            return api.delete({
+                version: 'v2',
+                url: `${BASE_URL}/${accountId}`,
+            }, callback);
+        },
+
+        uploadAccountDoc(accountId, params, callback) {
+
+            if (!accountId) {
+                return Promise.reject("Account Id is mandatroy");
```

Comment:
Question : Do we usually validate params in client? Why do we have validation only for this method?

Reply:
@1995navinkumar We only validate the id, Is this the right way to do it or should we remove it ?

Reply:
Wouldn't the API return an error if account id is not passed? If yes, should we just throw the error which API returns?

Reply:
@dhwanilvyas yes its manadatory in all also if the account id is not passed then api throw `The requested URL was not found on the server.`